src/router_core/delivery.c

```diff
@@ -503,6 +501,11 @@ void qdr_delivery_link_peers_CT(qdr_delivery_t *in_dlv, qdr_delivery_t *out_dlv)
     if (!in_dlv || !out_dlv)
         return;
 
+    qdr_link_t *link = qdr_delivery_link(in_dlv);
+    if (link) {
+        qd_log(link->core->log, QD_LOG_TRACE, DLV_FMT" :in qdr_delivery_link_peers_CT out: "DLV_FMT, DLV_ARGS(in_dlv), DLV_ARGS(out_dlv));
```

Comment:
Instead of 
qd_log(link->core->log, QD_LOG_TRACE, DLV_FMT" :in qdr_delivery_link_peers_CT out: "DLV_FMT, DLV_ARGS(in_dlv), DLV_ARGS(out_dlv));
can we say
qd_log(link->core->log, QD_LOG_TRACE, DLV_FMT" :in qdr_delivery_link_peers_CT out_dlv: "DLV_FMT, DLV_ARGS(in_dlv), DLV_ARGS(out_dlv));